### PR TITLE
Test/ui components

### PR DESCRIPTION
**Pull request by vyorkin**
I think this can merge merged into develop so we can write tests for a new components

### CODE REVIEW

File: packages/ui-components/src/internals/Button/__tests__/__snapshots__/index.test.tsx.snap [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+/>
+`;
+
+exports[`<Button /> renders correctly 2`] = `
+<button
+  class="root foo"
+/>
+`;
+
+exports[`<Button /> renders correctly 3`] = `
```
Comment (vyorkin) [inline packages/ui-components/src/internals/Button/__tests__/__snapshots__/index.test.tsx.snap]: this is how snapshots look like

File: packages/ui-components/src/internals/Button/__tests__/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import Button, { Props } from '..';
+
+describe('<Button />', () => {
+  it('renders correctly', () => {
+    const properties: Props[] = [
```
Comment (vyorkin) [inline packages/ui-components/src/internals/Button/__tests__/index.test.tsx]: test example

File: packages/ui-components/src/internals/Button/__tests__/__snapshots__/index.test.tsx.snap (hunk 1 above)
Comment (Bitaru) [reply packages/ui-components/src/internals/Button/__tests__/__snapshots__/index.test.tsx.snap]: woooow, lets do it!)

File: packages/ui-components/src/internals/Icon/index.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+export type MappedProps = {
+  width: number;
+  height: number;
+};
+
+export type Props = OwnProps & MappedProps;
+
+const svgLoader = '!!babel-loader!react-svg-loader!./icons/';
```
Comment (vyorkin) [inline packages/ui-components/src/internals/Icon/index.tsx]: this is the problem, need to stub it

File: packages/ui-components/src/internals/Image/__tests__/index.test.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      const component = render(<Image {...props} />);
+      expect(component).toMatchSnapshot();
+    });
+  });
+
+  it('removes "loading" class when loading is done', () => {
+    // TODO
+    const wrapper = shallow(<Image src="quux.jpg" />);
+    // expect(wrapper.hasClass('loading')).toBe(true);
+    wrapper.simulate('load');
```
Comment (vyorkin) [inline packages/ui-components/src/internals/Image/__tests__/index.test.tsx]: need to discover how to simulate those kind of events

File: packages/ui-components/src/widgets/Rating/index.tsx [hunk 5]
```diff
@@ -6,27 +6,32 @@ import Icon from 'internals/Icon';
...
+      {itemsArray.map((idx: number) => (
+        <Icon
+          key={idx}
+          name={`star-${getIconClass(idx - value)}`}
+          className={cx(styles.star)}
+        />
       ))}
     </div>
-    {!!count && <span className={styles.count}>({count})</span>}
+    {!count && <span className={styles.count}>({count})</span>}
```
Comment (vyorkin) [inline packages/ui-components/src/widgets/Rating/index.tsx]: its my mistake I think, will check

Comment (vyorkin) [reply packages/ui-components/src/widgets/Rating/index.tsx]: yeah, will fix that & merge

**Pull request merged**